### ISSUE TITLE
Invalid redirection URL

### ISSUE THREAD

**Issue by wantsoftat** [ollo69/ha-smartthinq-sensors#103]
after entering redirect url
![изображение](https://user-images.githubusercontent.com/40737183/96455364-cbf73d80-1225-11eb-94f0-2ad33ecaefc8.png)
![изображение](https://user-images.githubusercontent.com/40737183/96455215-96eaeb00-1225-11eb-9f8f-6fd32169a7b9.png)
until recently, everything worked

**Comment by KTibow** [comment]
Make sure you can use the SmartThinQ app on your phone, and try pasting in to the other field instead.

**Comment by ollo69** [comment]
Any progress here? Did you solve your issue?

**Comment by wantsoftat** [comment]
the mobile application works, the web page of the mobile site too. But url redirect throws error

**Comment by ollo69** [comment]
Are you using social network for login or a dedicated LG account?

**Comment by wantsoftat** [comment]
Dedicated LG account

**Comment by KTibow** [comment]
What happens if you paste the URL into the other field?

**Comment by wantsoftat** [comment]
Recorded my actions 
[https://drive.google.com/file/d/1RP3qbdbkEhUuK2D4Un8iQmCVUrAu0GGL/view?usp=sharing](url)

**Comment by ollo69** [comment]
Something you can try:

1) on the phone app logout and than login again, maybe you will be also asked to agree new TOS. After this try the procedure in HA again (like issue #95 and #101)
2) Try using a social network instead. In some case this solved the issue ([#89](https://github.com/ollo69/ha-smartthinq-sensors/issues/89#issuecomment-699491759))

If you still don't solve, enable debug for component and post here your HA log file.

**Comment by NorDroN** [comment]
When I try to login via facebook I see 
![image](https://user-images.githubusercontent.com/5458276/97446196-30f61600-193f-11eb-8f97-2ab9a2273e45.png).

**Comment by KTibow** [comment]
@NorDroN 
> If you still don't solve, enable debug for component and post here your HA log file.

**Comment by NorDroN** [comment]
I can't get correct Redirection URL, because I can't login. May be problem in Login URL, because I can login via wideq, but I cannot with this url.
Please, share your template of Redirection URL.

**Comment by NorDroN** [comment]
I don't try to hijack this thread, but I see that some people has problem with Redirection URL. I think that will be great if you provide example of correct Redirection URL in docs.

**Comment by kzoran** [comment]
Hi,
i'm in the US and i have same issue with re-direction URL, it just times out after i enter my username/password

**Comment by KTibow** [comment]
@kzoran you're supposed to copy the URL in the browser. Please open a new issue, people!

**Comment by wantsoftat** [comment]
Today I tried to log in again. And a miracle happened - everything worked.
Perhaps this is due to the LG servers?

**Comment by dustardly** [comment]
Same problem to me from Italy , maybe an update on LG server ?

**Comment by wantsoftat** [comment]
> 
> 
> Same problem to me from Italy , maybe an update on LG server ?

Try it in a few days - it helped me

**Comment by KTibow** [comment]
Again, update your LG ThinQ app on your phone, open it, and make sure it works.

**Comment by adamchengtkc** [comment]
I had the same issue. One day my washer integration stopped working and log says login refused. I tried reinstall and reconfig the integration again. Invalid credentials. Then I check the app. Turns out there is a new Terms of Service I need to agree. I accepted that and then all works fine.
Perhaps we need a way to catch this if LG require us accepting a new ToS before allowing login sequence to complete
